prog-o-meter.py

```diff
@@ -1,390 +1,419 @@
+        Window contains a submit button, for user to click when they have typed their name.
+        It does not matter if user types their name with Capital letter, fully capitalized, or in all lower letters. The name will always be in all lower letters in the name of the text file, where the user's data is stored.
+        The user's name will be displayed in a greeting in the window instantiated by the ProgressGUI class, and the format of the name their will be exactly as they typed it in this window.
+        That means that if the user types "Linea", first time they use the program, and "linea" second time they use it, the program will open the same .txt storing the data both times, but their name will be displayed differently each time.
+        """
+        # Attributes
+        self.root = Tk.Tk()
+        self.root.title("Login")
+        self.username = ""
+        self.user_type = _user_type
+        # Tkinter instantiation
+        self.canvas_layout()
+        self.input_button()
+        self.root.protocol("WM_DELETE_WINDOW", lambda: quit())
+        self.root.mainloop()
+    def canvas_layout(self):
+        """Display a Tkinter canvas.
+
+        Creates a 300x50 px canvas with a greeting text and an entry widget (input field for text).
+
+        Attributes:
+            CANVAS_WIDTH: The width of the canvas is hardcoded to 300 px.
+            CANVAS_HEIGHT: The height of the canvas is hardcoded to 50 px.
+            canvas: The Tkinter canvas widget.
+            text_entry: A Tkinter text-entry widget (input field for text)
+        """
+        self.CANVAS_WIDTH = 300
+        self.CANVAS_HEIGHT = 50
+        self.canvas = Tk.Canvas(self.root, width = self.CANVAS_WIDTH, height = self.CANVAS_HEIGHT)
+        self.canvas.pack()
+        self.text_entry = Tk.Entry(self.root)
+        self.text_entry.pack()
+        self.text_entry.focus_force()
+        if self.user_type == 1:        # Display appropriate greeting for returning users
+            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text = "Good to see you again! Please enter your name")
+        elif self.user_type == 2:        # Display appropriate greeting for new users
+            self.canvas.create_text(self.CANVAS_WIDTH/2, 20, text = "Lets get you started! Please enter your name")
+    def input_button(self):
+        """Display the inout button on the canvas.
+
+        Displays a submit button, for user to click when they have typed in their name.
+        When button is clicked, it stores the input name as username, and then closes the Tkinter window.
+
+        Attributes:
+            submit_button: Button with the text "Submit", which calls the function save_and_close
+        """
+        self.submit_button = Tk.Button(self.root, text = "Submit", command = self.save_and_close)
+        self.return_button = Tk.Button(self.root, text = "Go back to start", command = lambda: return_to_start(self.root))
```

Comment:
I think we could maybe have the text just be "cancel". Instead of "go back to start"